mt7615/mcu.c

```diff
@@ -190,7 +190,7 @@ int mt7615_mcu_parse_response(struct mt76_dev *mdev, int cmd,
 		ret = *skb->data;
 		break;
 	case MCU_EXT_CMD_GET_TEMP:
-		skb_pull(skb, sizeof(*rxd));
+		skb_pull(skb, sizeof(*rxd) + 4 * is_mt7663(mdev));
```

Comment:
```suggestion
		skb_pull(skb, sizeof(*rxd) + is_mt7663(mdev)?4:0);
```
A bit more readable/robust imho...